src/quicknxs/interfaces/data_handling/quicknxs_io.py

```diff
@@ -31,10 +31,10 @@
     "low_res_width": "y_width",
     "bck_position": "bg_pos",
     "bck_width": "bg_width",
-    "do_final_rebin_global": "g_final_rebin",
-    "final_rebin_step_global": "g_Qsteps",
-    "do_final_rebin_run": "r_final_rebin",
-    "final_rebin_step_run": "r_Qsteps",
+    "binning_type_global": "g_final_rebin",
```

Comment:
Won't this give problem when loading an "old" reduced file, meaning what where the types of `g_final_rebin` and `r_final_rebin` before you introduced `binning_q_step_global` and `binning_q_step_run`?